profiles/serializer/normalizer.py

```diff
@@ -0,0 +1,38 @@
+from collections import OrderedDict
+from functools import singledispatch
+from typing import Any
+
+from profiles.models import Profile
+
+
+@singledispatch
+def normalize(value: Any) -> str:
```

Comment:
Not sure of a good Python pattern for normalising/denormalising JSON to/from an object.

Reply:
I'm in the dark too